### ISSUE TITLE
Using dragon a user cannot  “Open this example in a new tab”

### ISSUE THREAD

**Issue by CharlotteDowns** [alphagov/govuk-design-system#2853]
## What
We received a 'bonus' piece of advice related to the accessibility of the GOV.UK Design System website. 

On our visit to the Digital Accessibility Centre (DAC) to observe our audit of GOV.UK Frontend components to WCAG 2.2 we received feedback on the 'Open example in a new tab' containers from our website. They highlighted a probable fail under [WCAG 2.5.3 "Label in name"](https://www.w3.org/WAI/WCAG22/Understanding/label-in-name.html).

## Why
In our code we have some `govuk-visually-hidden` text that appears between "Open this" and "example in a new tab". A user using voice assistive technologies would have their request not recognised as they are unaware of the visually hidden text in between.

## Screenshots
![Screenshot of the inspector view alongside the link to "Open this example in a new tab" showing the visually hidden text](https://github.com/alphagov/govuk-design-system/assets/56260216/d5cd187e-afec-4f85-904c-fd23d9c09b49)

## Possible solution
Have the visually hidden text appear after "Open this example in a new tab".

**Comment by davidc-gds** [comment]
My suggestion would be to have it read "Open this example in a new tab: [example label]"

So for example, for the first code example on the [back link component page](https://design-system.service.gov.uk/components/back-link/):
> Open this example in a new tab: back link
